src/pages/workspace/accounting/intacct/advanced/SageIntacctPaymentAccountPage.tsx

```diff
@@ -0,0 +1,76 @@
+import variables from '@styles/variables';
+import {updateSageIntacctSyncReimbursementAccountID} from '@userActions/connections/SageIntacct';
+import * as Policy from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function SageIntacctPaymentAccountPage({policy}: WithPolicyConnectionsProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const policyID = policy?.id ?? '-1';
+
+    const {config} = policy?.connections?.intacct ?? {};
+
+    const vendorSelectorOptions = useMemo<SelectorType[]>(() => getSageIntacctBankAccounts(policy, config?.sync?.reimbursementAccountID), [policy, config?.sync?.reimbursementAccountID]);
+
+    const updateDefaultVendor = useCallback(
+        ({value}: SelectorType) => {
+            if (value !== config?.sync?.reimbursementAccountID) {
+                updateSageIntacctSyncReimbursementAccountID(policyID, value);
+            }
+            Navigation.goBack(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_ADVANCED.getRoute(policyID));
+        },
+        [policyID, config?.sync?.reimbursementAccountID],
+    );
+
+    const listEmptyContent = useMemo(
+        () => (
+            <BlockingView
+                icon={Illustrations.TeleScope}
+                iconWidth={variables.emptyListIconWidth}
+                iconHeight={variables.emptyListIconHeight}
+                title={translate('workspace.common.noAccountsFound')}
+                subtitle={translate('workspace.common.noAccountsFoundDescription', CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT)}
+            />
+        ),
+        [translate],
+    );
+
+    return (
+        <SelectionScreen
+            policyID={policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            displayName={SageIntacctPaymentAccountPage.displayName}
+            sections={vendorSelectorOptions.length ? [{data: vendorSelectorOptions}] : []}
+            listItem={RadioListItem}
+            onSelectRow={updateDefaultVendor}
+            initiallyFocusedOptionKey={vendorSelectorOptions.find((mode) => mode.isSelected)?.keyForList}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_ADVANCED.getRoute(policyID))}
+            title="workspace.sageIntacct.paymentAccount"
+            listEmptyContent={listEmptyContent}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT}
+            pendingAction={config?.pendingFields?.reimbursementAccountID}
+            errors={ErrorUtils.getLatestErrorField(config ?? {}, CONST.SAGE_INTACCT_CONFIG.REIMBURSEMENT_ACCOUNT_ID)}
+            errorRowStyles={[styles.ph5, styles.mv2]}
+            onClose={() => Policy.clearSageIntacctErrorField(policyID, CONST.SAGE_INTACCT_CONFIG.REIMBURSEMENT_ACCOUNT_ID)}
+        />
+    );
+}
+
+SageIntacctPaymentAccountPage.displayName = 'PolicySageIntacctPaymentAccountPage';
```

Comment:
```suggestion
SageIntacctPaymentAccountPage.displayName = 'SageIntacctPaymentAccountPage';
```

Reply:
I prefer the displayName to be the same as the component name because when we have warnings or error logs, it will display as the displayName, so it will be easier to find 😄